README.md

```diff
@@ -132,11 +115,13 @@ Support for the PDC can be reached at https://bionimbus-pdc.opensciencedatacloud
 
 ## Commands
```

Comment:
Looks like we are missing the `configure` command. Perhaps the meat of `configure` can be put here and refer to it above.